ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -652,6 +655,12 @@ objects:
             exit 1
         fi
 
+        # TODO: Replace with a more concise manifest injection approach
+        if [[ -n "${CLUSTER_NETWORK_MANIFEST}" ]]; then
+            openshift-install --dir=/tmp/artifacts/installer/create manifests
+            echo "$CLUSTER_NETWORK_MANIFEST" > /tmp/artifacts/installer/manifests/cluster-network-03-config.yml
```

Comment:
Why is the one above "${CLUSTER_NETWORK_MANIFEST}" and this one "$CLUSTER_NETWORK_MANIFEST"?

If they are both "${CLUSTER_NETWORK_MANIFEST}" you don't need to declare the pod env var on the template.

Reply:
no particular reason, i just mixed syntax. Since I didn't put a template variable, I did not even think about this. I will amend, and remove the envvar.